src/mark.js

```diff
@@ -142,6 +149,39 @@ class Mark {
     }
 
     /**
+     * Sets up the regular expression string to allow later insertion of
+     * desinated characters (soft hyphens & zero width characters)
+     * @param  {string} str - The search term to be used
+     * @return {string}
+     * @access protected
+     */
+    setupIgnoreJoinersRegExp(str) {
+        // adding a "null" unicode character as it will not be modified by the
+        // other "create" regular expression functions
+        return str.replace(/[^(|)]/g, function(val, indx, original) {
+            // don't add a null after an opening "(", around a "|" or before
+            // a closing "("
+            let nextChar = original.charAt(indx + 1);
+            return /[(|)]/.test(nextChar) || nextChar === "" ? val : val + '\u0000';
```

Comment:
Please change this to:

```javascript
if(/[(|)]/.test(nextChar) || nextChar === ""){
    return val;
} else {
    return "${val}\u0000";
}
```

This will not overlap the max line length of 80 chars and uses `"` instead of `'`.